### DISCUSSION TITLE
Vibe's Figma & Storybook icons

### DISCUSSION THREAD

**Discussion by Ma1kovich** [mondaycom/vibe#2068] [General]
We want to use Monday Vibe's icons in our Figma, but we see that [Vibe's Figma](https://www.figma.com/file/pTpO3Wobs5cQwRsabt21Nd/monday.com-UI-kit?type=design&node-id=14812-8860&mode=design&t=reFX6l3sZCg1sKZY-0) contains much less icons then there are in the [package/storybook](https://style.monday.com/?path=/docs/media-icon--docs).
Are there any plans to keep them sync and up to date?

**Comment by talkor** [comment]
Thanks for bringing this up! You're correct—the current set of icons in the Figma library doesn't match the full range available in the package and Storybook. We're aware of this issue and definitely plan to sync them to ensure consistency across all tools. We'll surely update as it becomes available!
